pom.client.xml

```diff
@@ -1271,6 +1271,7 @@
     <module>sdk/keyvault/azure-security-keyvault-certificates</module>
     <module>sdk/keyvault/azure-security-keyvault-keys</module>
     <module>sdk/keyvault/azure-security-keyvault-secrets</module>
+	  <module>sdk/search/azure-search</module>
```

Comment:
`azure-search` dependency should also be added in `eng/jacoco-test-coverage/pom.xml` to ensure code coverage reports show up in our dashboard.

Reply:
done